hoodie-client/src/main/java/com/uber/hoodie/HoodieReadClient.java

```diff
@@ -57,24 +60,21 @@
    * just with a simple basepath pointing to the dataset. Until, then just always assume a
    * BloomIndex
    */
-  private transient final HoodieBloomIndex index;
+  private transient final HoodieIndex<T> index;
   private final HoodieTimeline commitTimeline;
-  private HoodieTable hoodieTable;
+  private transient HoodieTable hoodieTable;
```

Comment:
Any specific reason to make this transient ?

Reply:
This is transient in HoodieWriteClient, so this is just to align with HoodieWriteClient.

Reply:
actually I can revert the transient change for HoodieTable.